docs/logging-and-managing-experiment-results/logging-experiment-data.rst

```diff
@@ -0,0 +1,1587 @@
+
+Everything that is evaluated after ``neptune.create_experiment()`` and before the end of the script (or when you call :meth:`~neptune.experiments.Experiment.stop`) can be logged to the experiment.
+
+.. _what-you-can-log:
+
+-----
+
+What objects can you log to Neptune
+-----------------------------------
+Neptune supports logging many different types of data. Here, you can find all of them listed and described.
+
+.. note::
+
+    Remember to set project using :meth:`~neptune.init` and :meth:`~neptune.projects.Project.create_experiment` before you start logging.
+
+* :ref:`Metrics <logging-experiment-data-metrics>`
+* :ref:`Parameters <logging-experiment-data-parameters>`
+* :ref:`Code <logging-experiment-data-code>`
+
+    * :ref:`Git <logging-experiment-data-code-git>`
+    * :ref:`Code Snapshot <logging-experiment-data-code-code-snapshot>`
+    * :ref:`Notebook Snapshot <logging-experiment-data-code-notebook-snapshot>`
+
+* :ref:`Images <logging-experiment-data-images>`
+
+    * :ref:`Matplotlib <logging-experiment-data-images-matplotlib>`
+    * :ref:`PIL <logging-experiment-data-images-pil>`
+    * :ref:`NumPy <logging-experiment-data-images-numpy>`
+
+* :ref:`Interactive charts <logging-experiment-data-interactive-charts>`
+
+    * :ref:`Matplotlib <logging-experiment-data-interactive-charts-matplotlib>`
+    * :ref:`Altair <logging-experiment-data-interactive-charts-altair>`
+    * :ref:`Bokeh <logging-experiment-data-interactive-charts-bokeh>`
+    * :ref:`Plotly <logging-experiment-data-interactive-charts-plotly>`
+
+* :ref:`Text <logging-experiment-data-text>`
+* :ref:`Hardware consumption <logging-experiment-data-hardware-consumption>`
+* :ref:`Experiment information <logging-experiment-data-experiment-information>`
+
+    * :ref:`Experiment name <logging-experiment-data-experiment-information-name>`
+    * :ref:`Experiment description <logging-experiment-data-experiment-information-description>`
+    * :ref:`Experiment tags <logging-experiment-data-experiment-information-tags>`
+
+* :ref:`Properties <logging-experiment-data-properties>`
+* :ref:`Data versions <logging-experiment-data-data-versions>`
+* :ref:`Files <logging-experiment-data-files>`
+
+    * :ref:`Model checkpoints <logging-experiment-data-files-model-checkpoints>`
+    * :ref:`HTML objects <logging-experiment-data-files-html-objects>`
+
+* :ref:`Video <logging-experiment-data-video>`
+* :ref:`Audio <logging-experiment-data-audio>`
+* :ref:`Tables <logging-experiment-data-tables>`
+
+    * :ref:`pandas <logging-experiment-data-pandas>`
+    * :ref:`csv <logging-experiment-data-csv>`
+
+* :ref:`Python objects <logging-experiment-data-python-objects>`
+
+    * :ref:`Explainers (DALEX) <logging-experiment-data-python-objects-dalex>`
+    * :ref:`Pickled object <logging-experiment-data-python-pickle>`
+
```

Comment:
I think we should add integrations here for consistency -> it is mentioned down below but not here.

Reply:
added